test/suite/test_cursor18.py

```diff
@@ -157,14 +157,14 @@ def test_ondisk_with_deletion_on_update_chain(self):
 
         evict_cursor = self.session.open_cursor(self.uri, None, "debug=(release_evict)")
         self.session.begin_transaction()
-        self.assertEqual(evict_cursor[1], 0)
+        self.assertEquals(evict_cursor[1], 0)
```

Comment:
It now again uses assertEquals?

Reply:
Thanks -- that was a borked merge, I've fixed it since.